improve readme (#25)
google/temporian: README.md

![Temporian logo](resources/banner.png)

![tests](https://github.com/google/temporian/actions/workflows/test.yaml/badge.svg) ![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)

**Temporian** is a library to pre-process temporal signals before their use as input features with off-the-shelf tabular machine learning libraries (e.g., TensorFlow Decision Forests, scikit-learn).

## Usage Example

A minimal end-to-end run looks as follows:

```python

> import temporian as t

Reviewer: Update

![Temporian logo](resources/banner.png)

![tests](https://github.com/google/temporian/actions/workflows/test.yaml/badge.svg) ![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)

**Temporian** is a library to pre-process temporal signals before their use as input features with off-the-shelf tabular machine learning libraries (e.g., TensorFlow Decision Forests, scikit learn, XGBoost).

## Usage Example

A minimal end-to-end run looks as follows:

```python
import temporian as t

> # Load the data.

Reviewer 2: Remove fullstop after comment since no comments below have it

> event_data = t.read_event("path/to/data.csv")
> 
> # Event schema
> event = t.Event([t.Feature("sales", int), t.Feature("costs", int)])

Reviewer 2: ```suggestion
event_data = t.read_event("path/to/data.csv")
event = event_data.schema()
```

# Create Simple Moving Average feature
sma = t.simple_moving_average(
    input=event,
    window_length=t.day(5),
)

# Create Lag feature
lag = t.lag(
    input=event,
    lag=t.week(1),
)

# Assign features
output_event = t.assign(event, sma)
output_event = t.assign(output_event, lag)

> # Execute pre processing functions and get results

Reviewer 2: ```suggestion
# Execute pipeline and get results
```

output_event = t.evaluate(
    output_event,
    input_data={
        event: event_data,
    },
)

```

>__Warning__: The library is still under construction. This example usage is what we are aiming to build in the short term.

## Supported Features

> Temporian currently supports the following features for pre-processing your temporal data:

Reviewer 2: Some formatting of the markdown file seems off, `prettier` or a similar vscode extension should fix that for you

* **Simple Moving Average:** calculates the average value of each feature over a specified time window.

> * **Lag:** creates new features by shifting the time series data backward in time by a specified number of time steps.

Reviewer 2: ```suggestion
* **Lag:** creates new features by shifting the time series data backwards in time by a specified period.
```

> * **Leak:** creates new features by shifting the time series data forward in time by a specified number of time steps.

Reviewer 2: ```suggestion
* **Leak:** creates new features by shifting the time series data forward in time by a specified period.
```

* **Arithmetic Operations:** allows you to perform arithmetic operations (such as addition, subtraction, multiplication, and division) on time series data, between different events.

> More features comming soon...

Reviewer 2: ```suggestion
More features coming soon!
```

> ## Requirements for Contributors

Reviewer 2: ```suggestion
## Environment Setup
```

poetry shell
```

> ## Testing for Contributors

Reviewer 2: ```suggestion
## Testing
```
